src/analysis/retail/monk/mistweaver/modules/spells/SheilunsGiftCloudTracker.tsx

```diff
@@ -0,0 +1,13 @@
+import SPELLS from 'common/SPELLS';
+import { Options } from 'parser/core/Analyzer';
+import BuffStackTracker from 'parser/shared/modules/BuffStackTracker';
+
+export default class SheilunsGiftCloudTracker extends BuffStackTracker {
+  static trackPets = false;
+  static trackedBuff = SPELLS.SHEILUN_CLOUD_BUFF;
```

Comment:
Will this pick up SG cloud buffs from all mistweavers in a raid group?

Reply:
Nope, it's just for current player